shared-library/CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Contributing
+
+Fork this repository. Make your changes, tests it and provide a pull-request. That`s it.
+
+## Git workflow
```

Comment:
Since the plugin will setup library branch to release ID it is no more necessary.